benches/compute-digest.rs

```diff
@@ -54,7 +54,7 @@ where
     }
   }
 }
-impl<F> StepCircuit<F> for NonTrivialCircuit<F>
+impl<F> StepCircuit<F> for NonTrivialTestCircuit<F>
```

Comment:
FYI, this is just a renaming, but https://github.com/microsoft/Nova/commit/fb83e30e16e56b3b21c519b15b83c4ce1f077a13 makes it unlikely it will be accepted upstream. We should normalize to `NonTrivialCircuit`.